pkg/agent/route/route_linux.go

```diff
@@ -518,7 +518,7 @@ func (c *Client) AddRoutes(podCIDR *net.IPNet, nodeIP, nodeGwIP net.IP) error {
 
 	for _, route := range routes {
 		if err := netlink.RouteReplace(route); err != nil {
-			return fmt.Errorf("failed to install route to peer %s with netlink: %v", nodeIP, err)
+			return fmt.Errorf("failed to install route to peer %s with netlink. Route config: %s. Error: %v", nodeIP, route.String(), err)
```

Comment:
what's the current output when failing? I wonder what we are expecting from the change given the log already has peer and error.

Reply:
This is part of the log from a test. "err" is "invalid argument" but there's nothing more. In this case, I think knowing route config is helpful.
```
E0322 06:53:48.932631       1 node_route_controller.go:339] Error syncing Node tkg-scale-workload-md-0-d8b7b4cb4-b8ljv, requeuing. Error: failed to install route to peer fd00::203 with netlink: invalid argument
```

Reply:
Wonder what the new message is like?
Probably revise the log message and remove "peer" as it is included in the RouteConfig?

Reply:
I `klog.Infof()` the message without error.
```
failed to install route to peer 10.176.25.233 with netlink. Route config: {Ifindex: 6 Dst: 192.168.1.0/24 Src: <nil> Gw: 192.168.1.1 Flags: [onlink] Table: 0}
```

Reply:
It looks ok to me (might be better to change peer IP to Node name though).